src/platform/site-wide/sass/modules/_m-medallia.scss

```diff
@@ -206,11 +206,63 @@
 
 /* space between checkboxes on VAMC surveys */
 .neb-content .checkbox {
-  margin-bottom: 10px
+  margin-bottom: 10px !important;
 }
 
 /*  next button on VAMC muli-page surveys in-line style override*/
 .form-next-btn[style] {
   background-color: $color-blue !important;
   margin-right: 10px !important
 }
+
+a {
```

Comment:
Because this `a` and the `strong` below are not scoped (with a `.something` before them), won't these changes be applied to all `a` and `strong` tags throughout the website?

Reply:
@Andrew565 yes, good catch.  I changed both to scope only `.modal-live-form`